test/verify/check-menu

```diff
@@ -44,7 +44,10 @@ class TestMenu(MachineCase):
         b.click('#about button[data-dismiss="modal"]')
         b.wait_popdown("about")
 
-        self.check_axe("Test-navigation")
+        # HACK: We cannot test axe on Firefox since `axe.run()` returns promise
+        # and Firefox CDP cannot wait for it to resolve
+        if b.cdp.browser != "firefox":
+            self.check_axe("Test-navigation")
```

Comment:
Please put this hack into the `check_axe()` definition itself, instead of into every call (well, two right now, but we'll get more)